src/ai/susi/server/api/aaa/PasswordChangeService.java

```diff
@@ -113,18 +116,29 @@ public ServiceResponse serviceImpl(Query post, HttpServletResponse response, Aut
                 DAO.log("password change for user: " + identity.getName() + " via newpassword from host: " + post.getClientHost());
 
                 String subject = "Password Change";
+                InetAddress address = null;
+				try {
+					address = InetAddress.getLocalHost();
+				} catch (UnknownHostException e1) {
+					
+				}
+                String findIP = address.getHostAddress();
+                String IP = "127.0.1.1";
                 try {
                     EmailHandler.sendEmail(authentication.getIdentity().getName(), subject, "Your password has been changed successfully!");
                     result.put("message", "Your password has been changed!");
                     result.put("accepted", true);
                 } catch (Exception e) {
-                    throw new APIException(500, "Failed to change password");
+                    if(findIP.equals(IP)) {
+                    	result.put("message","Your password has sucessfully been changed and stored in the local database!");
+                    	result.put("accepted",true);
+                }
```

Comment:
please keep the Excpetion in the else case so we still have a hint that something did not work